ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -301,7 +271,7 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal, TwoLineTitl
         }
     }
 
-    var titleView = LargeTitleView() {
+    var titleView = AvatarTitleView() {
```

Comment:
This change seems unrelated to tokenizing. Should this be done in a separate PR? Additionally, what is the rationale for moving from LargeTitleView to AvatarTitleView?

Reply:
`LargeTitleView` is a bit of a misnomer now since it can also host a `TwoLineTitleView`. The defining characteristics of this view are that (a) it can show an avatar, and (b) its titles are leading-aligned. I chose the first one since the avatar is definitely an attention-grabber in practice.

It's tangentially related since we're tokenizing this component, so I think now's a fine time to rename it.

Reply:
Just to clarify - is it DIRECTLY related, or TANGENTALLY related? If it's tangental, as in, not very related, I really would rather it be in a separate PR.

In general, keeping PRs small and targeted is a good choice. Especially with a PR as big as this one has become, it might have been good to do this rename separately, after the large functionality change.

Reply:
It's relevant here because of the addition of `AvatarTitleViewTokenSet`, and as per my previous comment, `LargeTitleViewTokenSet` is an unsuitable name.

If you absolutely insist on this being a separate PR, see #1686, but reverting it here just to immediately change it back after this PR gets merged into the feature branch is just plain silly.